internal/pkg/template/workload.go

```diff
@@ -627,6 +627,11 @@ func envControllerParameters(o WorkloadOpts) []string {
 		}
 		parameters = append(parameters, "Aliases,") // YAML needs the comma separator; resolved in EnvContr.
 	}
+	if o.WorkloadType == "Backend Service" {
```

Comment:
I can already hear in my head @dannyrandall asking for a `switch` statement

Reply:
lol :joy: they are prettier :sob: